app/views/how-tos/index.html

```diff
@@ -14,13 +14,33 @@
 {% block content %}
   <div class="nhsuk-grid-row">
 
-    <div class="nhsuk-grid-column-two-thirds">
+    <div class="nhsuk-grid-column-full">
```

Comment:
Keep this as 2/3rds to limit line length of the paragraph below I think?

Reply:
Turns out there's a different way to do this in the design system:

````
<div class="nhsuk-grid-row">
  <div class="nhsuk-grid-column-full">
    <div class="nhsuk-u-reading-width">
      <p>This is example content which would exceed 70-80 characters per line, if used within a full width column. The .nhsuk-u-reading-width utility class will apply a maximum width and limit the number of characters per line.</p>
    </div>
  </div>
</div>
````